website/docs/using-plugins.md

```diff
@@ -275,6 +275,11 @@ module.exports = {
          * Whether to display the last date the doc was updated.
          */
         showLastUpdateTime: false,
+        /**
+         * Skip the next release docs when versioning is enabled.
+         * This will not build HTML files for documents in `/docs/next` directory, only versioned docs.
+         */
+        skipNextRelease: false,
```

Comment:
The name seems a bit confusing to me.

What about `excludeNextVersionDocs: true`?

Also worth documenting that this only appliest to build, not both build/start (and maybe the name of the option should reflect that?)

Reply:
Agree, fixed now.